runner/operator/argos_qc_operator/v1_1_0/construct_argos_qc_outputs.py

```diff
@@ -212,7 +212,7 @@ def get_output_directory_prefix(run_id_list):
 def get_project_prefix(run_id_list):
     project_prefix = set()
     for single_run_id in run_id_list:
-        project_prefix_port = Port.objects.filter(run=single_run_id, name="project_prefix")
+        project_prefix_port = Port.objects.filter(run=single_run_id, name="project_prefix")[0]
```

Comment:
Instead of `Port.objects.filter(run=single_run_id, name="project_prefix")[0]`, using `first()` is more optimal
```
Port.objects.filter(run=single_run_id, name="project_prefix").first()
```